pkg/test/framework/components/istio/config.go

```diff
@@ -149,6 +149,12 @@ type Config struct {
 
 	// Indicates that the test should deploy Istio's using helm charts
 	DeployHelm bool
+
+	// Version of the Istio control plane to deploy for this instance
+	Version string
+
+	// Revision for this control plane installation
+	Revision string
```

Comment:
Just a thought ... would it make sense to make this an enumeration rather than a free-text string? It might be easier from a testing perspective to just use `N_PLUS_1` or something like that.

Reply:
Yup, agreed. Done